PollBuddy-Server/frontend/src/pages/GroupJoin/GroupJoin.js

```diff
@@ -43,7 +63,13 @@ class GroupJoin extends Component {
         <MDBContainer fluid className="box">
           { this.state.showConfirm
             ?
-            <MDBContainer className="form-group">
+              <MDBContainer className="form-group">
+              <p className="fontSizeLarge">
+                Name: {this.state.name}
+              </p>
+              <p className="fontSizeLarge">
+                Description: {this.state.description}
+              </p>
```

Comment:
Lets have the Name and Description to look like this.

![image](https://user-images.githubusercontent.com/44654743/200129034-2ae2b30f-455d-4963-9282-6718ebcd2eea.png)